src/spatial/math.js

```diff
@@ -761,14 +770,20 @@ Crafty.math.Matrix2D = (function () {
      *
      * @public
      * @sign public {Matrix2D} new Matrix2D();
-     * @sign public {Matrix2D} new Matrix2D(Matrix2D);
-     * @sign public {Matrix2D} new Matrix2D(Number, Number, Number, Number, Number, Number);
-     * @param {Matrix2D|Number=1} a
+     * @returns {Matrix2D} A new identity matrix
+     * 
+     * @sign public {Matrix2D} new Matrix2D(Matrix2D matrix);
+     * @param {Matrix2D} matrix - a matrix to copy
+     * @returns {Matrix2D} A new instance whose entries are copied from the passed matrix
+     * 
+     * @sign public {Matrix2D} new Matrix2D(Number a, Number b, Number c, Number d, Number e, Number f);
+     * @param {Number=1} a
```

Comment:
I believe it was intended to indicate a default value.  That seemed useful enough that I modified the api generator to parse that out, but it doesn't get displayed anywhere yet by the docs.